workflow/v1/README.md

```diff
@@ -0,0 +1,167 @@
+present for all currently supported event types.
+
+<tr id="workflow.repository"><td><code>workflow.repository</code><td>string<td>
+
+HTTPS URI of the git repository, with `https://` protocol and without `.git`
+suffix.
+
+Can be computed from the [github context] using
+`github.server_url + "/" + github.repository`.
+
+<tr id="workflow.path"><td><code>workflow.path</code><td>string<td>
+
+The path to the workflow YAML file within the commit.
+
+Can be computed from the [github context] using `github.workflow_ref`, removing
+the prefix `github.repository + "/"` and the suffix `"@" + github.ref`. Take
+care to consider that the path and/or ref MAY contain `@` symbols.
+
+</table>
+
+[github context]:
+  https://docs.github.com/en/actions/learn-github-actions/contexts#github-context
+[release body parameters]:
+  https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28#create-a-release--parameters
+[variables]: https://docs.github.com/en/actions/learn-github-actions/variables
+[vars context]:
+  https://docs.github.com/en/actions/learn-github-actions/contexts#vars-context
+
+### Internal parameters
+
+All internal parameters are OPTIONAL.
+
+| Parameter | Type   | Description                                                                                                                                                               |
+| --------- | ------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `github`  | object | A subset of the [github context] as described below. Only includes parameters that are likely to have an effect on the build and that are not already captured elsewhere. |
+
+The `github` object SHOULD contains the following elements:
+
+| GitHub Context Parameter     | Type   | Description                                                                                             |
+| ---------------------------- | ------ | ------------------------------------------------------------------------------------------------------- |
+| `github.event_name`          | string | The name of the event that triggered the workflow run.                                                  |
+| `github.repository_id`       | string | The numeric ID corresponding to `externalParameters.workflow.repository`.                               |
+| `github.repository_owner_id` | string | The numeric ID of the user or organization that owns `externalParameters.workflow.repository`.          |
+| `github.runner_environment`  | string | The type of runner on which the the workflow was executed. One of the `github-hosted` or `self-hosted`. |
+
+Numeric IDs are used here to provide stable identifiers across account and
+repository renames and to detect when an old name is reused for a new entity.
+
+### Resolved dependencies
+
+The `resolvedDependencies` SHOULD contain an entry identifying the resolved git
```

Comment:
I think we should update the language here to MUST, as we reject provenance that does not have the `resolvedDependencies` field set.